src/bp/ui-studio/src/web/views/OneFlow/diagram/nodes/PromptContents/index.tsx

```diff
@@ -7,24 +8,46 @@ import style from '../Components/style.scss'
 
 interface Props {
   node: BlockModel
+  defaultLang: string
   selectedNodeItem: () => { node: BlockModel; index: number }
-  getCurrentLang: () => string
+  currentLang: string
 }
 
-const PromptContents: FC<Props> = ({ node, selectedNodeItem, getCurrentLang }) => {
-  const currentLang = getCurrentLang()
+const PromptContents: FC<Props> = ({ node, selectedNodeItem, currentLang }) => {
   const selectedContent = selectedNodeItem()
   const { next } = node || {}
   const { params } = node.prompt || {}
 
+  const fieldHasMissingTranslation = (value = {}) => {
```

Comment:
I'm not a fan of having pretty much this same function duplicated for Say & Prompts. Would be great if you could extract this

Reply:
me neither, but since it was temporary I wanted to just make it as quickly as possible so there might be some dupplicates